doc/manual/manual.tex

```diff
@@ -2780,8 +2776,8 @@ \subsubsection{Compiling \aspect{} and generating documentation}
 \end{verbatim}
 on the command line (or \texttt{make -jN} if you have multiple processors in
 your machine, where \texttt{N} is the number of processors). This builds the
-\aspect{} executable which will reside in
-the main directory and will be named \texttt{./aspect}. If you intend to
+\aspect{} executable which will reside in the \textt{build} directory
+and will be named \texttt{./aspect}. If you intend to
```

Comment:
the `./` is now no longer true. It'll be `./build/aspect`.

Are there other references to `./aspect` in the manual?